{"name": "test_div[get_datas_div0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <pratice_demo.pratice_calculator_advanced.test_calc_new.TestCalculator object at 0x00000227C9CF74F0>\nget_calc = <pratice_demo.pratice_calculator_advanced.demo.calc.Calculator object at 0x00000227C9B9A520>, get_datas_div = [2, 0, 0]\n\n    @allure.story(\"计算器的除法测试\")\n    @pytest.mark.run(order=4)\n    def test_div(self, get_calc, get_datas_div):\n        # 假设最大能输入999999999\n        if get_datas_div[0] > 999999999 or get_datas_div[1] > 999999999:\n            print(\"超出最大数值999999999，不允许输出\")\n            assert False\n        # 假设最小能输入-999999999\n        elif get_datas_div[0] < -999999999 or get_datas_div[1] < -999999999:\n            print(\"超出最小数值-999999999，不允许输出\")\n            assert False\n        # 判断除数是否为0，为0不能除\n        elif get_datas_div[1] == 0:\n            print(\"b为除数，不可以为0\")\n>           assert False\nE           assert False\n\ntest_calc_new.py:110: AssertionError"}, "attachments": [{"name": "stdout", "source": "4a0bf422-0860-4c1c-a183-80edf21639a0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_datas_div", "value": "[2, 0, 0]"}], "start": 1612345233810, "stop": 1612345233811, "uuid": "a7c9a493-c4bb-4fae-9ec3-7dd0363c2c7c", "historyId": "c3ffcf38930fc12944ebceab49168c9b", "testCaseId": "0134be1d683a1f54a7e80c7783a2eb76", "fullName": "test_calc_new.TestCalculator#test_div", "labels": [{"name": "feature", "value": "测试计算器"}, {"name": "story", "value": "计算器的除法测试"}, {"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_calc_new"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "Naraku15"}, {"name": "thread", "value": "6112-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc_new"}]}